src/config.rs

```diff
@@ -1542,33 +1565,83 @@ impl DBConfigManger {
                  block-cache.capacity in storage module instead"
                 .into());
         }
-        let opt = self.db.get_options_cf(cf)?;
-        opt.set_block_cache_capacity(size.0)?;
+        let mut result = Ok(());
+        self.tablet_factory
```

Comment:
Cache is also shared.

Reply:
When the cache is shared, it won't go to this code path ( there's a shortcut above)

Reply:
Cache must be shared.  Making every tablet use its owned cache will not work in practice. `shared_block_cache` is used for controlling cache between raft db and kv db in the past, not for tablets.

Reply:
As discussed offline, if multirocks enabled, the shared cache has to be enabled (which is the default now)---we would need to add the validation once the new flag is ready.
So the loop_tablet_cache code here only will run for single rocks db, which is totally fine.